packages/web/src/components/search/SearchBox.test.js

```diff
@@ -5,12 +5,42 @@ import SearchBox from './SearchBox';
 
 const MOCK_HITS_DATA = [
```

Comment:
I think you should maintain a different variable for featured suggestions mock data. You've changed the common mock data, which could be the reason of failing snapshots.